tests/linux/test_platform_linux.cpp

```diff
@@ -554,4 +555,172 @@ INSTANTIATE_TEST_SUITE_P(
+                         {"HOME_URL=\"https://www.ubuntu.com/\""},
+                         {"SUPPORT_URL=\"https://help.ubuntu.com/\""},
+                         {"BUG_REPORT_URL=\"https://bugs.launchpad.net/ubuntu/\""},
+                         {"PRIVACY_POLICY_URL=\"https://www.ubuntu.com/legal/terms-and-policies/privacy-policy\""},
+                         {"VERSION_CODENAME=hirsute"},
+                         {"UBUNTU_CODENAME=hirsute"}};
+
+    auto expected = std::pair<std::string, std::string>("A", "B");
+
+    auto output = mp::platform::parse_os_release(input);
+
+    EXPECT_EQ(expected.first, output.first.toStdString());
+    EXPECT_EQ(expected.second, output.second.toStdString());
+}
+
+TEST_F(PlatformLinux, parse_os_release_ubuntu2104lts)
+{
+    QStringList input = {{"NAME=\"Ubuntu\""},
+                         {"VERSION=\"21.04 (Hirsute Hippo)\""},
+                         {"ID=ubuntu"},
+                         {"ID_LIKE=debian"},
+                         {"PRETTY_NAME=\"Ubuntu 21.04\""},
+                         {"VERSION_ID=\"21.04\""},
+                         {"HOME_URL=\"https://www.ubuntu.com/\""},
+                         {"SUPPORT_URL=\"https://help.ubuntu.com/\""},
+                         {"BUG_REPORT_URL=\"https://bugs.launchpad.net/ubuntu/\""},
+                         {"PRIVACY_POLICY_URL=\"https://www.ubuntu.com/legal/terms-and-policies/privacy-policy\""},
+                         {"VERSION_CODENAME=hirsute"},
+                         {"UBUNTU_CODENAME=hirsute"}};
+
+    auto expected = std::pair<std::string, std::string>("Ubuntu", "21.04");
+
+    auto output = mp::platform::parse_os_release(input);
+
+    EXPECT_EQ(expected.first, output.first.toStdString());
+    EXPECT_EQ(expected.second, output.second.toStdString());
+}
+
+TEST_F(PlatformLinux, parse_os_release_ubuntu2104lts_rotation)
+{
+    QStringList input = {{"VERSION=\"21.04 (Hirsute Hippo)\""},
+                         {"ID=ubuntu"},
+                         {"ID_LIKE=debian"},
+                         {"VERSION_ID=\"21.04\""},
+                         {"PRETTY_NAME=\"Ubuntu 21.04\""},
+                         {"HOME_URL=\"https://www.ubuntu.com/\""},
+                         {"SUPPORT_URL=\"https://help.ubuntu.com/\""},
+                         {"BUG_REPORT_URL=\"https://bugs.launchpad.net/ubuntu/\""},
+                         {"PRIVACY_POLICY_URL=\"https://www.ubuntu.com/legal/terms-and-policies/privacy-policy\""},
+                         {"VERSION_CODENAME=hirsute"},
+                         {"NAME=\"Ubuntu\""},
+                         {"UBUNTU_CODENAME=hirsute"}};
+
+    auto expected = std::pair<std::string, std::string>("Ubuntu", "21.04");
+
+    auto output = mp::platform::parse_os_release(input);
+
+    EXPECT_EQ(expected.first, output.first.toStdString());
+    EXPECT_EQ(expected.second, output.second.toStdString());
+}
+
+TEST_F(PlatformLinux, parse_os_release_ubuntu2104lts_delimiter)
```

Comment:
Idem: I don't think we need to support handling arbitrary delimiters. The least functionality we commit to, the less we have to maintain :smile:

Reply:
Make sense, updated.